app/pages/About.tsx

```diff
@@ -1,6 +1,46 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 
+import AboutBio, { BioProps } from '../components/AboutBio';
+
+const team: BioProps[] = [
```

Comment:
This is so much better. I need to get better at abstracting the data away instead of treating everything like a simple template.